stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/actions/data/sql/actions/SQLDatasourceUtils.java

```diff
@@ -71,6 +71,9 @@
  */
 public class SQLDatasourceUtils {
 
+    private static final String ORACLE_DATABASE_NAME = "Oracle";
+    private static final int ORACLE_CURSOR_TYPE = -10;
+
```

Comment:
Shall we move these two into the org.ballerinalang.nativeimpl.actions.data.sql.Constants

Reply:
I kept these here since they are not shared with any other class. Should we still move these out?